lib/itamae/resource.rb

```diff
@@ -41,7 +41,7 @@ def get_resource_class(method)
 
       def autoload_plugin_resource(method)
         begin
-          require "itamae/plugin/resource/#{method}"
+          require_relative "plugin/resource/#{method}"
```

Comment:
I'm worried about this change. It may break the behavior of loading the external gem plugin, right?